src/library_memfs.js

```diff
@@ -341,7 +341,7 @@ mergeInto(LibraryManager.library, {
         var contents = stream.node.contents;
         // Only make a new copy when MAP_PRIVATE is specified.
         if ( !(flags & {{{ cDefine('MAP_PRIVATE') }}}) &&
-              (contents.buffer === buffer || contents.buffer === buffer.buffer) ) {
+              (contents.buffer === buffer) ) {
```

Comment:
Yeah, I see what you mean. Another option is `Module['buffer']` which clarifies it's a global thing?